src/constants/index.ts

```diff
@@ -0,0 +1,14 @@
+export const NavData =[
+    {
+    title: 'Balance', value:'5000'
+},
+{
+    title: 'Bet', value:'500'
```

Comment:
Bet value should be zero
Balance should be Number too, don't use string to represent number